modules/json/shared/src/test/scala/gem/json/instances/AsterismJsonSpec.scala

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.json.instances
+
+import cats.tests.CatsSuite
+import gem.Asterism
+import gem.arb._
+import io.circe.testing.CodecTests
+import io.circe.testing.instances._
+
+final class AsterismJsonSpec extends CatsSuite {
+  import ArbAsterism._
+  import asterism._
+
+  checkAll("Asterism", CodecTests[Asterism].unserializableCodec)
```

Comment:
Why are tests done on `unserializableCodec` rather than `codec`?

Reply:
If you're working with Spark or Akka everything has to be serializable so circe checks instances for serializability by default. But we don't care about that.